media-source-respec.html

```diff
@@ -2512,6 +2514,9 @@
 
     <section id="htmlvideoelement-extensions">
       <h2>HTMLVideoElement Extensions</h2>
+      <p>The following section uses <a>VideoPlaybackQuality</a> and is OPTIONAL. There are several implementations of the current specification, but it is subject to change because some feel it needs further work. The community is actively investigating alternate designs to provide feedback on media playback quality. See <a href="https://wicg.github.io/media-playback-quality/">Media Playback Quality</a>.
```

Comment:
Thanks for adding this. Besides fixing publishDate + regenerating and merging (and fixing the revision history as a result), this looks good to me. Thanks for doing this, @jdsmith3000